sandstone/client/sandstone/components/filetreedirective/tests/filetree.directive.test.js

```diff
@@ -1,208 +1,421 @@
-describe('Filetree directive', function(){
-  var $compile;
-  var scope;
-  var element;
-  var httpBackend;
-  var rootScope;
-  var filesystemservice;
-  var isolateScope;
-  var dirs = [{
-        "filepath": "/home/saurabh/dir1",
-        "filename": "dir1",
-        "group": "saurabh",
-        "is_accessible": true,
-        "perm": "-rw-rw-r--",
-        "perm_string": "664",
-        "size": "4.0 KiB",
-        "type": "dir",
-        "children": []
-      }, {
-        "filepath": "/home/saurabh/dir2",
-        "filename": "dir2",
-        "group": "root",
-        "is_accessible": false,
-        "perm": "-rw-r--r--",
-        "perm_string": "644",
-        "size": "4.0 KiB",
-        "type": "dir",
-        "children": []
-      },
-      {
-        "filepath": "/home/saurabh/dir3",
-        "filename": "dir3",
-        "group": "saurabh",
-        "is_accessible": true,
-        "perm": "-rw-rw-r--",
-        "perm_string": "664",
-        "size": "4.0 KiB",
-        "type": "dir",
-        "children": []
-      },
-      {
-        "filepath": "/home/saurabh/dir4",
-        "filename": "dir4",
-        "group": "saurabh",
-        "is_accessible": true,
-        "perm": "-rw-rw-r--",
-        "perm_string": "664",
-        "size": "4.0 KiB",
-        "type": "dir",
-        "children": []
-      }];
-
-    var files = [{
-          "filepath": "/home/saurabh/file1",
-          "filename": "file1",
+describe('filetree.directive', function() {
```

Comment:
Description should be `sandstone.filetree` to show where the component lies in the module hierarchy, and the actual name of the component.